packages/desktop/src/main/index.ts

```diff
@@ -41,6 +49,14 @@ const createMainWindow = () => {
         width: 1200,
     });
 
+    // When the renderer finishes loading, send the current environment.
+    mainWindow.webContents.on("did-finish-load", () => {
```

Comment:
Where is this event sent? Do we still need to listen here for this for the `mainWindow` to know when the environment changed?

Reply:
oh whoops forgot to remove. Fixed in [586d76b](https://github.com/AllenInstitute/biofile-finder/pull/416/commits/586d76bc92f89fe875762b56a116527e5be3c6a8)